functions/model/tasks/tark/logwork.js

```diff
@@ -76,12 +77,22 @@ exports.logWork = function(request, response) {
                 CompletionDate: completiondate,
             };
             updateTask(logWorkInputJson, orgDomain, taskId);
+            watchers.forEach(element => {
+                sendMail(element, subjectMessage, htmlMessage);
+                console.log("mail sent!!!!!!!")
+            });
```

Comment:
It's not gonna work, watcher is not filled

Reply:
done